### PR TITLE
Add fpm-alpine variant

### PR DESCRIPTION
**Pull request by ncopa**
Depends on docker-library/php/pull/176

### CODE REVIEW

**Comment by brunoqc** [comment]
Any update on this? Alpine based images are a lot smaller.

**Comment by yosifkit** [comment]
I think there are a few bugs to resolve in the `php` image before we push it to child images (https://github.com/docker-library/php/pull/206#issuecomment-222552221, https://github.com/docker-library/php/issues/234).

**Comment by shouze** [comment]
ping @ncopa @yosifkit could you take a look at my attempt to fix https://github.com/docker-library/php/pull/206#issuecomment-222552221 with  https://github.com/docker-library/php/pull/237?

**Comment by shouze** [comment]
ping @ncopa https://github.com/docker-library/php/pull/245 review welcome ;)

File: fpm/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM php:5.6-fpm-alpine
+
+# install the PHP extensions we need
+RUN apk add --no-cache --virtual .build-deps \
+		autoconf gcc libc-dev make \
+		libpng-dev libjpeg-turbo-dev \
+	&& docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+	&& docker-php-ext-install gd mysqli opcache \
+	&& find /usr/local/lib/php/extensions -name '*.a' -delete \
+	&& find /usr/local/lib/php/extensions -name '*.so' -exec strip --strip-all '{}' \; \
```
Comment (tianon) [inline fpm/alpine/Dockerfile]: Given that PHP's own `Dockerfile` strips all the executables it installs, I wonder if it wouldn't be more appropriate for `docker-php-ext-install` to do so as well (certainly would be more appropriate IMO to do this there generally than to do so here as a special case).

**Pull request closed without merging**